cloudos/jobs/job.py

```diff
@@ -221,6 +229,13 @@ def convert_nextflow_to_json(self,
                              }
         else:
             revision_block = ""
+        if storage_mode == "lustre":
```

Comment:
Add warning here for the user to "Check if your workspace of CloudOS has lustre enabled, before using this option"

**Q:** What happens if the user chooses this and lustre is not available in their workspace? @cgpu to test and report

Reply:
That's a good question @cgpu . I also tested and what I found is, if the workspace does not have Lustre available, the job fails very early, with only this error message (nothing inside the job):
![Screenshot from 2022-06-29 19-42-22](https://user-images.githubusercontent.com/45285897/176501838-f642b96c-8552-426f-87f3-99c035c8b835.png)

Therefore, it's a very good idea to show a warning / message to the user :+1:

Reply:
Warning message added. I opted for a regular print, which in my opinion is less stresful than an actual Python warning:
![Screenshot from 2022-06-29 20-10-11](https://user-images.githubusercontent.com/45285897/176506511-e1be28cc-3321-4062-9930-71fd2d5db3dc.png)